src/complexity.cc

```diff
@@ -196,6 +213,10 @@ std::vector<BenchmarkReporter::Run> ComputeStats(
       cpu_accumulated_time_stat.Mean() * run_iterations;
   mean_data.bytes_per_second = bytes_per_second_stat.Mean();
   mean_data.items_per_second = items_per_second_stat.Mean();
+  // user counters
+  for(auto const& kv : counter_stats) {
+    mean_data.counters[kv.first.c_str()] = kv.second.Mean(); /**@todo add the rest of the settings (fmt,flags)*/
```

Comment:
are you going to do this in this PR?

if not, please watch the line length: move the todo comment to the line before.